Gemfile

```diff
@@ -25,8 +25,13 @@ gem 'pundit'
 gem 'valid_email'
 gem 'rack-attack'
 gem 'rqrcode'
-gem 'ruby-saml'
-gem 'saml_idp', '~> 0.3.1'
+
+# unreleased feature via: https://github.com/onelogin/ruby-saml/pull/345
+gem 'ruby-saml', github: 'onelogin/ruby-saml', branch: 'master'
+
+# upstream PR: https://github.com/sportngin/saml_idp/pull/51
+gem 'saml_idp', '~> 0.3.1', github: 'pkarman/saml_idp', branch: 'principal-asserted-attributes'
```

Comment:
I could. I would rather the PR just get merged and released upstream. If you like, we can wait for that to merge this PR, which means no need for comments in the Gemfile.